system/modules/isotope/library/Isotope/Model/Payment/Payone.php

```diff
@@ -168,4 +164,22 @@ function(&$option) {
 
         return $objTemplate->parse();
     }
+
+    /**
+     * Escape the parameter value so it's correctly handled by Payone
+     *
+     * @param string $value
+     *
+     * @return string
+     */
+    protected function escapeParameterValue($value)
+    {
+        $value = strip_insert_tags(strip_tags($value));
+        $value = \StringUtil::restoreBasicEntities($value);
+        $value = html_entity_decode($value);
+        $value = str_replace("\xC2\xA0", ' ', $value); // strip &nbsp;
+        $value = str_replace('"', '\'', $value); // replace double quotes with single quotes
```

Comment:
Absolutely not you mean 😉  The ```\'``` is escape only in PHP so it results as a single quote in HTML. You can't escape the form value as follows:

```html
<input type="hidden" name="foobar" value="my \"super\" product name">
```